test/faker/gov/us/ssn_test.exs

```diff
@@ -0,0 +1,34 @@
+defmodule Faker.SSN.EnUsTest do
```

Comment:
The module name does not match the filename. Also, I think the module name should be called `Faker.Gov.UsTest`, since it's a test for the `Faker.Gov.Us` module. The file should be called `test/faker/gov/us_test.exs`